components/usage-api/usage/v1/billing.proto

```diff
@@ -78,8 +78,9 @@ message CreateStripeCustomerResponse {
 
 message CreateStripeSubscriptionRequest {
   string attribution_id = 1;
-  string setup_intent_id = 2;
-  int64 usage_limit = 3;
+  string customer_id = 2;
```

Comment:
We're not using the `customer_id` in the implementation of `CreateStripeSubscription`, do we need it here?

FYI, Reordering fields (by changing their index) is a backwards incompatible change normally. Here, it doens't matter as we're not calling it yet.

Reply:
I changed the proto back - which is what we have on main.